src/together/cli/api/finetune.py

```diff
@@ -200,6 +200,19 @@ def fine_tuning(ctx: click.Context) -> None:
     "The format: {$JOB_ID/$OUTPUT_MODEL_NAME}:{$STEP}. "
     "The step value is optional, without it the final checkpoint will be used.",
 )
+@click.option(
+    "--from-hf-model",
+    type=str,
+    default=None,
```

Comment:
Nit: `None` is the default already https://github.com/pallets/click/blob/main/src/click/core.py#L2081